code/ui/manager/src/components/sidebar/TreeNode.stories.tsx

```diff
@@ -41,6 +41,43 @@ export const Expandable = () => (
   </>
 );
 
+export const ExpandableLongName = () => (
+  <>
+    <ComponentNode isExpandable>
```

Comment:
To test word wrapping/breaking I think you want a story with a single long word with out spaces?

I might be misinterpreting the changes.